src/components/ExtraInfo.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import { Signs, ExtraInfoHead, UnitMeasure } from '@utils/constants';
+import colors from '@styles/colors';
+
+export enum ExtraInfoType {
+    Precipitation,
+    Visibility,
+    RealFeel,
+    Humidity,
+}
+
+type CategoryToData = {
+    [key in ExtraInfoType]: IComponentData;
+};
+
+interface IComponentData {
+    src: number;
+    extroInfoTitle: string;
+    extroInfoComment: string;
+    additionalInfo?: string;
+    unitMeasure: string;
+}
+
+export interface IExtraInfoProps {
+    catergory: ExtraInfoType;
+    digitalValue: number;
+}
+
+const Icons: CategoryToData = {
+    [ExtraInfoType.Precipitation]: {
+        src: require('@assets/icons/water_drop_outline_20.png'),
+        extroInfoTitle: ExtraInfoHead.PRECIPITATION,
+        extroInfoComment: 'Ничего не ожидается в ближайшие 10 дней',
+        additionalInfo: 'за 24 часа',
+        unitMeasure: UnitMeasure.MILLIMETERS,
+    },
+    [ExtraInfoType.Visibility]: {
+        src: require('@assets/icons/view_outline_28.png'),
+        extroInfoTitle: ExtraInfoHead.VISIBILITY,
+        extroInfoComment: 'Видимость снижена из-за дыма',
+        unitMeasure: UnitMeasure.KILOMETERS,
+    },
+    [ExtraInfoType.RealFeel]: {
+        src: require('@assets/icons/thermometer.png'),
+        extroInfoTitle: ExtraInfoHead.REAL_FEEL,
+        extroInfoComment: 'Дождь может стать прохладнее',
+        unitMeasure: Signs.CELSIUS,
+    },
+    [ExtraInfoType.Humidity]: {
+        src: require('@assets/icons/fog_16.png'),
+        extroInfoTitle: ExtraInfoHead.HUMIDITY,
+        extroInfoComment: 'Точка росы\nсейчас: 8' + Signs.PERCENT,
```

Comment:
For the future it is not the best idea to break lines manually. If you dont like how lines auto break you can use nonbreaking space